CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.1.2
+
+- Support being run from outside a Bundler environment
+- Don't update the advisory DB twice when checking Github repos
```

Comment:
No, you're right. I thought I'd included it in the PR but again, my push failed and I didn't notice.